Tasks/AzureFileCopyV4/AzureFileCopy.ps1

```diff
@@ -118,8 +118,22 @@ try {
         if([string]::IsNullOrEmpty($containerName) -or ($destination -ne "AzureBlob"))
         {
             $containerName = [guid]::NewGuid().ToString()
-            Create-AzureContainer -containerName $containerName -storageContext $storageContext -isPremiumStorage $isPremiumStorage
+            Write-Verbose "Creating temporary container for uploading files if no input is provided for container name"
```

Comment:
this should be: Container Name input not found. Creating Temporary container for uploading files.